src/Agent.Worker/Container/ContainerInfo.cs

```diff
@@ -1,3 +1,4 @@
+using Agent.Sdk;
 using System;
```

Comment:
this file may cause conflicts with some changes @fadnavistanmay is doing. You may want to revert this one file.